website/mails/mail_callbacks.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+
+from website import settings
+
+def no_addon(email):
+    email.data['fullname'] = email.user.fullname
```

Comment:
Did you mean to leave this in? `fullname` is passed in to `queue_mail` so it should already be in `email.data`